### PR TITLE
NAS-104418 / 12.0 / Nas 104418

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 1]
```diff
@@ -159,9 +160,8 @@ export const helptext_sharing_iscsi = {
   extent_validators_name: [Validators.required],
 
   extent_placeholder_type: T("Extent type"),
-  extent_tooltip_type: T(
-    "<i>File</i> shares the contents of an individual file.\
- <i>Device</i> shares an entire device."),
+  extent_tooltip_type: T("<i>Device</i> provide virtual storage access to zvols, zvol snapshots, or physical devices.\
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
  extent_tooltip_type: T("<i>Device</i> provides virtual storage access to zvols, zvol snapshots, or physical devices. \
```

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 2]
```diff
@@ -159,9 +160,8 @@ export const helptext_sharing_iscsi = {
...
+  <i>File</i> provides virtual storage access to an individual file."),
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
  <i>File</i> provides virtual storage access to a single file."),
```

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 3]
```diff
@@ -317,5 +315,44 @@ export const helptext_sharing_iscsi = {
...
+
+  // wizard
+  step1_label: T("Create or Choose Block Device"),
+
+  name_placeholder: T("Name"),
+  name_tooltip: T("Keep the name short. Using a name longer than 63 characters can prevent accessing the block device."),
+
+  disk_placeholder: T("Device"),
+  disk_tooltip: T("Select the unformatted disk, controller, zvol, or zvol snapshot. Select\
+ <i>Create New</i> for options to create a new zvol."),
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
 <i>Create New</i> to create a new zvol."),
```

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 4]
```diff
@@ -317,5 +315,44 @@ export const helptext_sharing_iscsi = {
...
+  dataset_placeholder: T("Pool/Dataset"),
+  dataset_tooltip: T("Browse to an existing pool or dataset to store the new zvol."),
+
+  volsize_placeholder: T("Size"),
+  volsize_tooltip: T("Specify the size of the new zvol."),
+
+  volblocksize_placeholder: T("Block Size"),
+  volblocksize_tooltip: T("Only override the default if the initiator requires a different block size."),
+
+  usefor_placeholder: T("What are you using this for"),
```
Comment (Mrt134) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
  usefor_placeholder: T("What are you using this for?"),
```
I don't particularly like this placeholder, but I can't come up with a solid alternative in the 10-20 seconds I have available to think about this. "Platform for Share", "Sharing Platform", eh

Comment (lola27) [reply src/app/helptext/sharing/iscsi/iscsi.ts]: Thanks @Mrt134, changed it to 'Sharing Platform'.

**Pull request merged**